sessions/006/README.md

```diff
@@ -326,14 +326,9 @@ pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b   20Gi       RWO            Delete
 pvc-1a66039a-14e7-4416-9319-6d6437543c02   20Gi       RWO            Delete           Bound    test/data-my-cluster-kafka-0                                         ocs-external-storagecluster-ceph-rbd            18m
 pvc-ca348d35-f466-446e-9f93-e3c15722d214   20Gi       RWO            Delete           Bound    test/data-my-cluster-kafka-2                                         ocs-external-storagecluster-ceph-rbd            18m
 ```
-
-
-=====
-
-### ZOOKEEPER
-
+<br/>
```

Comment:
Add an empty line after this, otherwise rendering is screwed up.

Reply:
Oh, checking.